cvmfs/s3fanout.cc

```diff
@@ -636,8 +637,9 @@ int S3FanoutManager::InitializeDnsSettings(
   }
 
   // Remove port number if such exists
```

Comment:
The comment seems to be wrong.  We want to make sure that the protocol is part of the URL.